src/context/framework.provider.tsx

```diff
@@ -0,0 +1,67 @@
+import { ComponentType, createContext, ReactNode, useContext, useMemo } from "react";
+
+export enum Framework {
+  PPC = "ppc",
+  TF = "terrafund",
+  TF_LANDSCAPES = "terrafund-landscapes",
+  ENTERPRISES = "enterprises",
+  HBF = "hbf",
+
+  UNDEFINED = "undefined"
+}
+
+export const ALL_TF = [Framework.TF, Framework.TF_LANDSCAPES, Framework.ENTERPRISES];
```

Comment:
Previous code in some cases defined Terrafund as "not HBF or PPC", so including the three TF frameworks in one array makes replacing code like that simple.